static/app/views/admin/options.tsx

```diff
@@ -174,6 +174,38 @@ const performanceOptionDefinitions: Field[] = [
     max: Number.MAX_SAFE_INTEGER,
     step: 1,
   },
+  {
+    key: 'performance.issues.consecutive_db.problem-creation',
+    label: t('N+1 API Calls Problem Creation Rate'),
+    help: t(
+      'Controls the overall rate at which performance problems are detected by the N+1 API Calls detector.'
+    ),
+    ...HIGH_THROUGHPUT_RATE_OPTION,
+  },
+  {
+    key: 'performance.issues.consecutive_db.la-rollout',
+    label: t('Limited Availability Detection Rate'),
+    help: t(
+      'Controls the rate at which performance problems are detected by the N+1 API Calls detector for LA organizations.'
+    ),
+    ...HIGH_THROUGHPUT_RATE_OPTION,
+  },
```

Comment:
Looks like you forgot to change the label and help text for the new options

Reply:
huhh, yes I did lol, thanks @0Calories

Reply:
Fixed!